pkgs/misc/ghostscript/default.nix

```diff
@@ -149,6 +149,10 @@ stdenv.mkDerivation rec {
 
   # validate dynamic linkage
   doInstallCheck = true;
+  preInstallCheck = if stdenv.hostPlatform.isDarwin then ''
+    DYLD_LIBRARY_PATH=$out/lib
+    export DYLD_LIBRARY_PATH
+  '' else null;
```

Comment:
I believe this will still cause rebuilds (unless it defaults to `null`?). You generally need `${if stdenv.hostPlatform.isDarwin then "preInstallCheck" else null} = …`.

Reply:
ofborg thinks it's fine.